lib/system/generic/xlnx/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/microblaze_generic)
+  add_subdirectory(microblaze_generic)
+else()
+  collect (PROJECT_LIB_SOURCES sys.c)
+endif()
+
+collect (PROJECT_LIB_HEADERS sys.h)
+
+collect (PROJECT_LIB_SOURCES irq.c)
+
+# vim: expandtab:ts=2:sw=2:smartindent
```

Comment:
remove this line

Reply:
which line? the line 9 is required for xlnx platforms
@arnopo

Reply:
line 11:` # vim: expandtab:ts=2:sw=2:smartindent`